lib/snappable.dart

```diff
@@ -34,6 +34,9 @@ class Snappable extends StatefulWidget {
   /// Defaults to false
   final bool snapOnTap;
 
+  /// Function that gets called when snap ends
+  final Function onSnapped;
```

Comment:
Could you change the type to VoidCallback to be more accurate? :)

Reply:
Done!